app/config/config.yml

```diff
@@ -49,16 +49,19 @@ doctrine:
         auto_mapping: true
 
 swiftmailer:
-    transport: '%mailer_transport%'
-    host: '%mailer_host%'
-    username: '%mailer_user%'
-    password: '%mailer_password%'
+    transport: gmail
```

Comment:
amend this commit. sensitive data must not be commited, especially to public repository!